intezer_sdk/_endpoint_analysis_api.py

```diff
@@ -0,0 +1,97 @@
+                                                                  method='POST')
+        response.raise_for_status()
+
+    def send_injected_modules_info(self, injected_module_list: dict):
+        response = self.request_with_refresh_expired_access_token(path='/injected-modules-info',
+                                                                  data=injected_module_list,
+                                                                  method='POST')
+        response.raise_for_status()
+
+    def send_scheduled_tasks_info(self, scheduled_tasks_info: dict):
+        response = self.request_with_refresh_expired_access_token(path='/scheduled-tasks-info',
+                                                                  data=scheduled_tasks_info,
+                                                                  method='POST')
+        response.raise_for_status()
+
+    def send_file_module_differences(self, file_module_differences: dict):
+        response = self.request_with_refresh_expired_access_token(path='/file-module-differences',
+                                                                  data=file_module_differences,
+                                                                  method='POST')
+        response.raise_for_status()
+
+    def send_files_info(self, files_info: dict) -> List[str]:
+        """
+        :param files_info: endpoint scan files info
+        :return: list of file hashes to upload
+        """
+        response = self.request_with_refresh_expired_access_token(path='/files-info',
+                                                                  data=files_info,
+                                                                  method='POST')
+        response.raise_for_status()
+        return response.json()['result']
+
+    def send_memory_module_dump_info(self, memory_modules_info: dict) -> List[str]:
+        """
+        :param memory_modules_info: endpoint scan memory modules info
+        :return: list of file hashes to upload
+        """
+        response = self.request_with_refresh_expired_access_token(path='/memory-module-dumps-info',
+                                                                  data=memory_modules_info,
+                                                                  method='POST')
+        response.raise_for_status()
+        return response.json()['result']
+
+    def upload_collected_binary(self, file_path: str, collected_from: str):
+        with open(file_path, 'rb') as file_to_upload:
+            file_data = file_to_upload.read()
+            compressed_data = zlib.compress(file_data, zlib.Z_BEST_COMPRESSION)
+            response = self.request_with_refresh_expired_access_token(
+                path=f'/{collected_from}/collected-binaries',
+                body=compressed_data,
+                headers={'Content-Type': 'application/octet-stream', 'Content-Encoding': 'gzip'},
```

Comment:
Not sure if you tried it, but there might be a way to use data or files instead of adding another body parameter